fiftyone/core/fields.py

```
@@ -1513,14 +1513,18 @@ def to_mongo(self, value):
         if value is None:
             return None
 
+        if isinstance(value, str):
+            value = fou.deserialize_numpy_array(value, ascii=True)
+
         bytes = fou.serialize_numpy_array(value)
```

Ruff (0.15.10)

[error] Variable `bytes` is shadowing a Python builtin

(A001)